Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -2886,6 +2886,19 @@ namespace AssetProcessor
 
     void AssetProcessorManager::AssessFilesFromScanner(QSet<AssetFileInfo> filePaths)
     {
+        if (m_initialScanSkippingFeature)
+        {
+            for (const AssetFileInfo& fileInfo : filePaths)
+                AddKnownFoldersRecursivelyForFile(fileInfo.m_filePath, fileInfo.m_scanFolder->ScanPath());
+
+            m_sourceFilesInDatabase.clear();
+            m_fileModTimes.clear();
+            m_fileHashes.clear();
```

Comment:
I see you added a fast scan button.  Will that button function if this data is cleared?  I didn't see anything to re-populate it.

Reply:
Yes, those maps are actually only needed while a scan is performed and gets cleared once the scan finishes.

We tested the fast scan button without any issue.